### DISCUSSION TITLE
Looking for larger datasets

### DISCUSSION THREAD

**Discussion by hua0x522** [deepmodeling/deepmd-kit#2476] [Q&A] [answered]
Hello, I have noticed that some papers related to DeePMD-kit use large amounts of data(such as reaching 100 million atoms in this paper **_86 PFLOPS Deep Potential Molecular Dynamics simulation of 100 million atoms with ab initio accuracy_**). May I inquire how one could obtain such a large dataset?

<img width="538" alt="dataset" src="https://user-images.githubusercontent.com/68761626/233890917-533a73a2-f7b1-481f-a8b5-14882ad4c230.PNG">

**Comment by AnuragKr** [comment]
For MD Simulation we can create a larger number of atoms using the _replicate_ command in LAMMPS.
You may want to look into this discussion - https://github.com/deepmodeling/deepmd-kit/discussions/1964

**Comment by njzjz** [comment, marked as answer]
We have published these two systems at https://github.com/deepmodeling-activity/deepmd-kit-v2-paper. As answered by @AnuragKr, you have to use the `replicate` command for these systems.
